zarr/tests/test_hierarchy.py

```diff
@@ -912,6 +919,17 @@ def create_store():
         store = ZipStore(path)
         return store, None
 
+    def test_context_manager(self):
+
+        with self.create_group() as g:
+            store = g.store
+            d = g.create_dataset('foo/bar', shape=100, chunks=10)
+            d[:] = np.arange(100)
+
+        error = ValueError if sys.version_info >= (3, 6) else RuntimeError
+        with pytest.raises(error):
+            store.zf.extractall()
```

Comment:
Just checking, is this testing that the underlying ZipStore has been closed and hence the zip file is closed? If so, might be useful to add a comment that this is what this is doing.

Reply:
Exactly. I'll do that.